includes/gateways/paypal/scripts.php

```diff
@@ -13,6 +13,40 @@
 
 use EDD\PayPal\Exceptions\Authentication_Exception;
 
+/**
+ * Enqueues polyfills for Promise and Fetch.
+ *
+ * @since 2.11
+ */
+function maybe_enqueue_polyfills() {
+	/**
+	 * Filters whether or not IE11 polyfills should be loaded.
+	 * Note: This filter may have its default changed at any time, or may entirely
+	 * go away at one point.
+	 *
+	 * @since 2.11
+	 */
+	if ( ! apply_filters( 'edd_load_ie11_polyfills', true ) ) {
+		return;
+	}
+
+	wp_enqueue_script(
+		'promise-polyfill',
+		EDD_PLUGIN_URL . 'assets/js/promise-polyfill.min.js',
+		array(),
+		'8.2.0',
+		true
+	);
+
+	wp_enqueue_script(
+		'fetch-polyfill',
+		EDD_PLUGIN_URL . 'assets/js/fetch-polyfill.min.js',
+		array( 'promise-polyfill' ),
+		'3.6.0',
+		true
+	);
```

Comment:
@ashleyfae WordPress 5.0+ ships with polyfills for these that could be used if on a supported version: https://github.com/WordPress/gutenberg/blob/086b77ed409a70a6c6a6e74dee704851eff812f2/docs/contributors/code/scripts.md#polyfill-scripts

Reply:
Brilliant - thanks @spencerfinnell !